application/frontend/src/pages/LinkEntity.tsx

```diff
@@ -42,7 +42,7 @@ export function LinkEntity() {
             key: 'entityType',
         },
         {
-            title: 'Delete',
+            title: 'Link',
```

Comment:
what is the purpose of this change @batuhanovski ?

Reply:
it was just an old correction :D It is not related to "deadline feature".

Reply:
OK 👍